--- gcc-4.6-2012.03/gcc/cp/cfns.gperf   2012-03-29 10:01:56.000000000 +0200
+++ gcc-4.6-2012.03.new/gcc/cp/cfns.gperf       2023-11-10 21:54:01.174531739 +0100
@@ -19,8 +19,2 @@
-#ifdef __GNUC__
-__inline
-#endif
-static unsigned int hash (const char *, unsigned int);
-#ifdef __GNUC__
-__inline
-#endif
-const char * libc_name_p (const char *, unsigned int);
+static unsigned int hash (const char *, long unsigned int);
+const char * libc_name_p (const char *, long unsigned int);
